src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java

```diff
@@ -54,6 +64,44 @@ NetworkInfos importCase(UUID caseUuid) {
         return new NetworkInfos(networkUuid, network.getId());
     }
 
+    ExportNetworkInfos exportCase(UUID networkUuid, String format) throws IOException {
+        if (!Exporters.getFormats().contains(format)) {
+            throw NetworkConversionException.createFormatUnsupported(format);
+        }
+        MemDataSource memDataSource = new MemDataSource();
+        Network network = networkStoreService.getNetwork(networkUuid);
+        Exporters.export(format, network, null, memDataSource);
+
+        Set<String> listNames = memDataSource.listNames(".*");
+        String networkName;
+        byte[] networkData;
+        if (listNames.size() == 1) {
+            networkName = network.getNameOrId() + listNames.toArray()[0];
+            networkData = memDataSource.getData(listNames.toArray()[0].toString());
+        } else {
+            networkName = network.getNameOrId() + ".zip";
+            networkData = createZipFile(listNames.toArray(new String[0]), memDataSource).toByteArray();
```

Comment:
i didn't understand that instruction : listNames.toArray(new String[0]) ! why new String[0]

Reply:
to convert the list to a String array